src/applications/yellow-ribbon/components/SearchResult/index.jsx

```diff
@@ -42,51 +38,60 @@ const deriveLocationLabel = (school = {}) => {
 };
 
 const deriveMaxAmountLabel = (school = {}) => {
-  // Show unknown if there's no tuitionOutOfState.
-  if (!school?.tuitionOutOfState) {
+  // Show unknown if there's no contributionAmount.
+  if (!school?.contributionAmount) {
     return 'Unknown';
   }
 
-  // Show formatted tuitionOutOfState.
-  return school?.tuitionOutOfState.toLocaleString('en-US', {
+  // Derive the contribution amount number.
+  const contributionAmountNum = parseFloat(school?.contributionAmount);
+
+  if (contributionAmountNum > 90000) {
+    return 'All tuition and fees not covered by Post-9/11 GI Bill';
+  }
```

Comment:
This is a little confusing to me. If the school contributes over `90000`, then `All tuition and fees not covered by Post-9/11 GI Bill`?

Reply:
That's what the spec said, it's confusing to me too: https://vsateams.invisionapp.com/share/P4W04QPWHTK#/screens/405289243

Reply:
Let's ping @andaleliz and @AnneHurley, they may have some ideas!

Reply:
We're trying to replace the word "unlimited" because it's not really "unlimited".  A student wouldn't get endless amounts of money.

Does that help? If not, can you please re-phrase the question? Thanks!

Reply:
Oh I see...I read that like the prospective student wouldn't be fully covered. Something like `All tuition and fees after Post-9/11 GI Bill funding` might've helped but I'm biased now 🤷‍♂

Reply:
@ncksllvn thanks for clarifying, I see what you're saying. Perhaps @AnneHurley has suggestions for word smithing. If not, I say we just leave it as-is.